### DISCUSSION TITLE
About WebRTC Statistics

### DISCUSSION THREAD

**Discussion by voluntas** [paullouisageneau/libdatachannel#1032] [Q&A] [answered]
I would like to propose in the Pull-Request a mechanism to get WebRTC statistics. are there any negatives to getting WebRTC statistics into libdatachannel?

We would like to comply with the following
https://www.w3.org/TR/webrtc-stats/

If there are no negatives, we hope to implement this slowly and gradually.

**Comment by paullouisageneau** [comment, marked as answer]
WebRTC statistics would be a nice addition! However, note that it is not possible to implement everything as libdatachannel does not handle media encoding/decoding internally.

**Comment by voluntas** [comment]
Thanks for the reply. I will consider a small minimum first!
